src/components/common/Table.js

```diff
@@ -77,13 +77,19 @@ const Table = (props) => {
           const { key, className } = row;
           return (
             <TableRow key={key} className={className}>
-              {columns.map(({ field, alignField }) => {
-                return (
-                  <TableCell align={alignField} key={field}>
-                    {row[field]}
-                  </TableCell>
-                );
-              })}
+              {columns
+                .filter(({ field }) => field)
```

Comment:
Is there a case where `field` is undefined? Won't it shift the columns if you filter it?

Reply:
Yes, I wanted to allow a cell to span two columns with `colSpan` but then we need another column to occupy 0 columns which I do by filtering out.

This is used in `Validation.js` where the Validation result and status are defined with a single cell spanning over both of them.